processor/error/processor.go

```diff
@@ -20,12 +20,12 @@ const (
 func NewProcessor() pr.Processor {
 	schema := pr.CreateSchema(errorSchema, processorName)
 	return &processor{
-		&Payload{},
+		&payload{},
 		schema}
 }
 
 type processor struct {
-	payload *Payload
+	payload pr.Payload
```

Comment:
Just saw your comment on this. +1 on fixing the inconsistency, but i would go with the private payload in both cases.